react_components/PaginationBoxView.js

```diff
@@ -207,6 +208,7 @@ export default class PaginationBoxView extends Component {
           <a onClick={this.handlePreviousPage}
              className={this.props.previousLinkClassName}
              href={this.hrefBuilder(this.state.selected - 1)}
+             rel={this.state.selected !== 0 ? "prev" : null}
```

Comment:
Would it be better to say something like `previous page` rather than `prev`? Its better to be more descriptive.

Reply:
If I get this right, `previous page` wrong value for this attribute.
>Google treats rel="previous" as a syntactic variant of rel="prev"

https://support.google.com/webmasters/answer/1663744?hl=en